scripts/compare-dumps.go

```diff
@@ -72,9 +89,6 @@ func compare(a, b string) error {
 		if blockA.Block != blockB.Block {
 			return fmt.Errorf("block number mismatch: %d vs %d", blockA.Block, blockB.Block)
 		}
-		if blockA.Size != blockB.Size {
```

Comment:
We can recalculate sizes during normalization.

Reply:
Normalized size is len(block.Storage) - this check is present just several lines below.